tests/utils/test_fmrisim.py

```diff
@@ -592,6 +593,92 @@ def test_generate_noise():
                        )
 
 
+def test_generate_noise_spatial():
+
+    # Set up the inputs
+    dimensions = np.array([10, 5, 10])
+    mask = np.ones(dimensions)
+    vol = sim._generate_noise_spatial(dimensions, mask)
+
+    # Run the analysis from _calc_FHWM but for th elast step of aggregating
+    # across dimensions
+    v_count = 0
+    v_sum = 0
+    v_sq = 0
+
+    d_sum = [0.0, 0.0, 0.0]
+    d_sq = [0.0, 0.0, 0.0]
+    d_count = [0, 0, 0]
+
+    # Pull out all the voxel coordinates
+    coordinates = list(product(range(dimensions[0]),
+                               range(dimensions[1]),
+                               range(dimensions[2])))
+
+    # Find the sum of squared error for the non-masked voxels in the brain
+    for i in list(range(len(coordinates))):
+
+        # Pull out this coordinate
+        x, y, z = coordinates[i]
+
+        # Is this within the mask?
+        if mask[x, y, z] > 0:
+
+            # Find the the volume sum and squared values
+            v_count += 1
+            v_sum += np.abs(vol[x, y, z])
```

Comment:
If the goal is to calculate the variance, I think `np.abs` is not necessary here.